src/app/api/schemas.py

```diff
@@ -137,7 +137,7 @@ class DeviceIn(DefaultPosition):
 
 
 class DeviceAuth(DeviceIn):
```

Comment:
Maybe remove the password field and inherit from Cred?

Reply:
done!